algorithm/data-structure/LinkedList/SourceCode/LinkedListSatu.java

```diff
@@ -0,0 +1,16 @@
+import java.util.LinkedList;
+
+public class LinkedListSatu {
+    public static void main(String[] args) {
```

Comment:
```suggestion
    public static void main(String[] args) throws Exception {
```

coba test biar detect errornya